node/cmd/guardiand/node.go

```diff
@@ -705,16 +731,25 @@ func runNode(cmd *cobra.Command, args []string) {
 		if err := supervisor.Run(ctx, "oasiswatch",
 			ethereum.NewEthWatcher(*oasisRPC, oasisContractAddr, "oasis", common.ReadinessOasisSyncing, vaa.ChainIDOasis, lockC, nil, 1, chainObsvReqC[vaa.ChainIDOasis]).Run); err != nil {
 			return err
-		}
+		}				
+
 		if *testnetMode {
 			if err := supervisor.Run(ctx, "ethropstenwatch",
-				ethereum.NewEthWatcher(*ethRopstenRPC, ethRopstenContractAddr, "ethropsten", common.ReadinessEthRopstenSyncing, vaa.ChainIDEthereumRopsten, lockC, setC, 1, chainObsvReqC[vaa.ChainIDEthereumRopsten]).Run); err != nil {
+				ethereum.NewEthWatcher(*ethRopstenRPC, ethRopstenContractAddr, "ethropsten", common.ReadinessEthRopstenSyncing, vaa.ChainIDEthereumRopsten, lockC, nil, 1, chainObsvReqC[vaa.ChainIDEthereumRopsten]).Run); err != nil {
```

Comment:
Are you setting the `setChan` to nil here because we should only have it set in ethwatch to get the gst?

https://github.com/certusone/wormhole/blob/107d697f4298dc668d6d7d866b4dae390342d916/node/pkg/ethereum/watcher.go#L80-L89

I don't know if we want to do this or not. @leoluk ^